test/utils/index.js

```diff
@@ -38,7 +38,8 @@ export function getSastAnalysis(
       return this.sourceFile.dependencies;
     },
     execute(body) {
-      const probeRunner = new ProbeRunner(this.sourceFile, [probe]);
+      const probes = Array.isArray(probe) ? probe : [probe];
```

Comment:
When does `getSastAnalysis` get an array of probes?

Reply:
i have removed this unecessary part, it was me playing arround with the code while initially debugging the issue, thanks